src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs

```diff
@@ -74,10 +75,15 @@ overrideHaskellPackages oc packages = vcat
     , "};"
     , ""
     ]
-  , "in callPackage <nixpkgs/pkgs/development/haskell-modules> {"
+  , "in callPackage (nixpkgsPath + \"/pkgs/development/haskell-modules\") {"
```

Comment:
Mostly because it gets used in two places, and I was planning to add another PR that uses the specified `_optNixpkgsRepository` to set this.